src/styles/stories/Margins/MarginExample.tsx

```diff
@@ -0,0 +1,49 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+
+import './MarginExample.scss';
+
+export enum MarginSize {
+  EXTRASMALL = 'xs',
+  SMALL = 'sm',
+  MEDIUM = 'md',
+  LARGE = 'lg',
+  EXTRALARGE = 'xl',
+}
+
+export interface MarginExampleProps
+  extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
In this and Padding story you don't need to extend HTML element interface for your pros if you are not going to provide any props to the HTML element.
Ask me about this offline if you didn't get it

Reply:
Discussed offline